app/src/main/java/com/dongyang/android/youdongknowme/ui/view/splash/SplashActivity.kt

```diff
@@ -51,6 +66,23 @@ class SplashActivity : AppCompatActivity() {
         }
     }
 
+    override fun onRequestPermissionsResult(
+        requestCode: Int, permissions: Array<String>, grantResults: IntArray
+    ) {
```

Comment:
```suggestion
        requestCode: Int,
        permissions: Array<String>,
        grantResults: IntArray
    ) {
```

줄바꿈이 있으면 더욱 가독성이 좋아질 것 같아요!

Reply:
변경하겠습니다!